opus/application/static_media/js/search.js

```diff
@@ -360,38 +360,99 @@ var o_search = {
 
         // range behaviors and string behaviors for search widgets - qtype select dropdown
         $('#search').on("change", "select", function() {
-            let qtypes = [];
-
-            switch ($(this).attr("class")) {  // form type
-                case "RANGE":
+            let isInputSetEmpty = true;
+            if ($(this).attr("name").startsWith("qtype-")) {
+                let qtypes = [];
+                let counterStr = o_utils.getSlugOrDataTrailingCounterStr($(this).attr("name"));
+                let idx = counterStr ? counterStr - 1 : 0;
+                switch ($(this).attr("class")) {  // form type
+                    case "RANGE":
```

Comment:
I'm not sure why this wasn't done before, but the code after each "case" should be indented one level.